### ISSUE TITLE
Overview time line graph on x-ray is empty when upgrade.

### ISSUE THREAD

**Issue by hey-jude** [metabase/metabase#9235]
### Bugs

Overview time line graph on x-ray is empty when I upgraded metabase from 0.30.4 to 0.31.2.

Refer attached images.

xray overview 0.30.4
<img width="1327" alt="xray_overview_0 30 4" src="https://user-images.githubusercontent.com/6929275/50826101-20e20080-137e-11e9-839b-f8df198663d4.png">

xray overview 0.31.2
<img width="1313" alt="xray_overview_0 31 2" src="https://user-images.githubusercontent.com/6929275/50826107-23445a80-137e-11e9-9dea-8f124a6a3be2.png">

===

- Your browser and the version: Chrome
- Your operating system: OS X
- Your databases: Druid 0.10.2
- Metabase version: 0.31.2
- Metabase hosting environment: Linux/Centos 7
- Metabase internal database: H2 (default)

**Comment by salsakran** [comment]
if you click into the empty graph, what is the underlying question defined as and what does it return?

**Comment by hey-jude** [comment]
@salsakran 
Mouse click is not entered, but drag empty graph can be.

<img width="893" alt="screen shot 2019-02-11 5 47 40" src="https://user-images.githubusercontent.com/6929275/52553130-f1fbf600-2e25-11e9-8b35-6b504011f4db.png">

**Comment by sbelak** [comment]
I can repo this on our stats instance. But it only happens when timestamp is aggregated by day, both hour and month are fine.

**Comment by mazameli** [comment]
Stacktrace from @davidllorente from https://github.com/metabase/metabase/issues/9690

```
Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged and will be removed in an upcoming major release. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
Arguments: 
[0] _isAMomentObject: true, _isUTC: false, _useUTC: false, _l: undefined, _i: 2019-03-28+01:00, _f: undefined, _strict: undefined, _locale: [object Object]
Error
    at Function.createFromInputFallback (http://localhost:3000/app/dist/vendor.bundle.js?7d6d729f0c3014d9a886:1:23741)
    at Mt (http://localhost:3000/app/dist/vendor.bundle.js?7d6d729f0c3014d9a886:1:41462)
    at Ct (http://localhost:3000/app/dist/vendor.bundle.js?7d6d729f0c3014d9a886:1:42173)
    at Tt (http://localhost:3000/app/dist/vendor.bundle.js?7d6d729f0c3014d9a886:1:42255)
    at Function.i.parseZone (http://localhost:3000/app/dist/vendor.bundle.js?7d6d729f0c3014d9a886:1:70441)
    at t.parseTimestamp (http://localhost:3000/app/dist/app-main.bundle.js?7d6d729f0c3014d9a886:5:274067)
    at t.HACK_parseTimestamp (http://localhost:3000/app/dist/app-main.bundle.js?7d6d729f0c3014d9a886:5:406992)
    at http://localhost:3000/app/dist/app-main.bundle.js?7d6d729f0c3014d9a886:5:400423
    at Array.map (<anonymous>)
    at http://localhost:3000/app/dist/app-main.bundle.js?7d6d729f0c3014d9a886:5:400318
```

**Comment by sbelak** [comment]
Closed in favour of #9690
